Automattic/woocommerce-payments
# Fix MultiCurrency onboarding settings page on Woo Express

Fixes #6466

#### Changes proposed in this Pull Request

This PR aims to fix the MultiCurrency onboarding settings page on Woo Express that was rendering the debug statistics due to broken HTML being output. The HTML was broken due to the lack of a `sectionend` with the same `id` as the `title` section. 

Proposed changes:
- Add a `sectionend` section
- Remove the `Save settings` button as it doesn't make sense on that page

The page content gets mixed with the MC settings without the `sectionend` section, which causes the debug tool to be rendered with the MC settings. After adding the new section, the settings page knows where to close the tags, and the page gets rendered correctly.









#### Testing instructions





**Before checking out to this branch**

* Go to **WCPay Dev** ([Docker link](http://localhost:8082/wp-admin/admin.php?page=wcpaydev)), check "Force the WCPay plugin to act as disconnected from the WCPay Server" and save the changes.
* Go to **Plugins > Add New** and search for "Debug bar" ([Docker link](http://localhost:8082/wp-admin/plugin-install.php?s=Debug%2520bar&tab=search&type=term)) – install and activate the plugin.
* Go to **WooCommerce > Settings** and select the "Multi-currency" tab ([Docker link](http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency)).
* Inspect the "Save changes" button and confirm that the `#querylist` div is inside the `#mainform` form
  * Expand the `#querylist` div on the developer tools and confirm that the debug bar HTML is there

<details><summary>Developer tools</summary>
<img src="https://github.com/Automattic/woocommerce-payments/assets/16882226/a4416267-3e03-477b-9425-bd28844063d6" />
</details> 

* Checkout to this branch
* Refresh the page
* Inspect the same element as before
* The form content should be contained inside the form tag and the debug bar should be far below on the DOM

<details><summary>Developer tools</summary>
<img src="https://github.com/Automattic/woocommerce-payments/assets/16882226/ae85712b-8a8a-4882-833c-5e55065a49cf" />
</details>

**Test it on the Woo Express site**

- Please, ping me to get the Woo Express site that is already configured with this WooPayments version.
- First check the fix
  - Go to **Settings > WooCommerce** and select the "Multi-Currency" tab
  - You should see the Multi-currency settings page correctly

<img width="2274" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/16882226/5a73b7e9-0c5b-4d83-88ea-3c255d16bd32">

- Check the current behavior (develop)
  - Go to **Plugins > Installed Plugins**
    - Deactivate **WooCommerce Payments Version 6.5.1-multicurrency-section-error**
    - Activate **WooCommerce Payments Version 6.5.1**
  - Go to **Settings > WooCommerce** and select the "Multi-Currency" tab
  - You should see the Multi-currency settings page and the debug bar content

![image](https://github.com/Automattic/woocommerce-payments/assets/16882226/a8a41592-4fdc-4831-b7a3-edf586b5eb62)

-------------------


**Post merge**
